src/displayapp/DisplayAppRecovery.cpp

```diff
@@ -83,8 +83,7 @@ void DisplayApp::DisplayLogo(uint16_t color) {
   for(int i = 0; i < displayWidth; i++) {
     rleDecoder.DecodeNext(displayBuffer, displayWidth * bytesPerPixel);
     ulTaskNotifyTake(pdTRUE, 500);
-    lcd.BeginDrawBuffer(0, i, displayWidth, 1);
-    lcd.NextDrawBuffer(reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * bytesPerPixel);
+    lcd.DrawBuffer(0, 0, displayWidth, displayHeight, reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * 2);
```

Comment:
this is wrong, must be
lcd.DrawBuffer(0, i, displayWidth, 1, reinterpret_cast<const uint8_t *>(displayBuffer), displayWidth * bytesPerPixel);